cypress/integration/version/task_filters.ts

```diff
@@ -56,7 +49,7 @@ describe("Tasks filters", () => {
         .find("input")
         .focus()
         .type(variantInputValue)
-        .type("{enter}");
+        .type("{enter}", { scrollBehavior: false });
```

Comment:
can we set `scrollBehavior` false for this whole test or do we need it in some places?

Reply:
I think its ok to keep it the way its intended. I found that the default scrollBehavior was only affecting these antd tooltips. They are so sensitive and disable if the window scrolls even by a little bit. 

We even have this ResizeObserver handling because of them. Once we move to the LG table we probably won't need this anymore.